Godeps/Godeps.json

```diff
@@ -9,35 +9,26 @@
 		{
 			"ImportPath": "bitbucket.org/ww/goautoneg",
 			"Comment": "null-5",
-			"Rev": "75cd24fc2f2c2a2088577d12123ddee5f54e0675"
+			"Rev": "'75cd24fc2f2c2a2088577d12123ddee5f54e0675'"
```

Comment:
Must be a change in godep binary - this is cleanly generated via `godep save` so just a tooling thing I guess.

Would like to change to a different vendoring tool like glide soon though tbh. I find godep pretty annoying to work with personally & would prefer the standard golang `vendor` dir anyway.